lib/sqlalchemy_ingres/base.py

```diff
@@ -387,6 +387,7 @@ class IngresDialect(default.DefaultDialect):
     supports_empty_insert = False
     supports_empty_insert = False
     supports_statement_cache = False  # NOTE this is not actually picked up by SA warning code, _generate_cache_attrs() checks dict of subclass, not the entire class
+    supports_schemas = False
```

Comment:
Recommend having a comment here, could simple say "see readme.testsuite.md for details"